sdk/compute/Azure.ResourceManager.Compute/src/Customize/VirtualMachineData.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
```

Comment:
We should remove this because this file is no longer auto generated.
Some our linter tools will detect the existence of this comment, and loosen the constraint of coding style, etc